src/main/webapp/auth.html

```diff
@@ -0,0 +1,44 @@
+<html>
+  <head>
```

Comment:
Already reviewed this previously. Why is it here as new change?

Reply:
Idk I have tried to delete the file from this branch and it gives me a 404 error